types/gavazzi/UWPA_UWPM/uplink.js

```diff
@@ -0,0 +1,66 @@
+// INT32 - Little Endian
+function INT32LE(hex, signed) {
+  const data = hex.match(/../g);
```

Comment:
why this regex?

Reply:
easy way to split the hex into an array

Reply:
what about hex.split("")?

Reply:
Does not work because i need to split the bytes (which are two characters long)
I could do a loop if you really dislike the regex

Reply:
ok no no you can leave it